src/content/en/updates/2018/05/webauthn.md

```diff
@@ -0,0 +1,164 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: Chrome 67 beta introduces the Web Authentication (WebAuthn) API, which allows browsers to interact with and manage public-key based credentials. This enables strong authentication using removable security keys and built-in platform authenticators such as fingerprint scanners.
+
+{# wf_updated_on: 2018-05-09 #}
+{# wf_published_on: 2018-05-10 #}
+{# wf_tags: webauthn, credentials, sign-in, chrome67 #}
+{# wf_featured_image: /web/updates/images/generic/security.png #}
+{# wf_featured_snippet: Chrome 67 beta introduces the Web Authentication (WebAuthn) API, which allows browsers to interact with and manage public-key based credentials. This enables strong authentication using removable security keys and built-in platform authenticators such as fingerprint scanners. #}
+{# wf_blink_components: Blink>WebAuthentication #}
+
+# Enabling strong authentication with WebAuthn {: .page-title }
+
+{% include "web/_shared/contributors/cbrand.html" %}
+{% include "web/_shared/contributors/agektmr.html" %}
+
+## The problem
+
+Phishing is the \#1 security problem on the web: 81% of account breaches last 
+year were because of weak or stolen passwords. The industry's collective 
+response to this problem has been multi-factor authentication, but 
+implementations are fragmented and most still don't adequately address phishing. 
+We have been working with the [FIDO Alliance](https://fidoalliance.org/) since 
+2013 and, more recently, with the W3C to implement a standardized 
+phishing-resistant protocol that can be used by any Web application.
+
+## What is WebAuthn?
+
+[The Web Authentication API](https://www.w3.org/TR/webauthn/) gives Web 
+applications user-agent-mediated access to authenticators – which are often 
+hardware tokens accessed over USB/BLE/NFC or modules built directly into the 
+platform – for the purposes of generating and challenging application-scoped 
+(eTLD+k) public-key credentials. This enables a variety of use-cases, such as:
+
+* Low friction and phishing-resistant 2FA (to be used in conjunction with 
+  password)
+* Passwordless, biometrics-based re-authorization
+* Low friction and phishing-resistant 2FA **without** password (to be used for 
+  passwordless accounts)
+
+The API is on track to be implemented by most major browsers, and is intended to 
+both simplify the UI encountered when having to prove your identity online and 
+significantly reduce phishing.
+
+WebAuthn extends the [Credential Management 
+API](https://w3c.github.io/webappsec-credential-management/) and adds a new 
```

Comment:
Should we point to our documentation on Cred Mgt API instead of the spec?

Reply:
I think this is fine.